invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -102,11 +103,16 @@ export class RDMDepositForm extends Component {
         files={files}
         permissions={permissions}
       >
-        <FormFeedback
-          fieldPath="message"
-          labels={this.config.custom_fields.error_labels}
-        />
-        <CommunityHeader imagePlaceholderLink="/static/images/square-placeholder.png" />
+        <Overridable id="InvenioAppRdm.deposit.FormFeedback.layout" {...this.props}>
```

Comment:
```
{...this.props}
```
this is a bit too broad, needs to be adapted to keep a clear component interface